independent-publisher-connectors/NIST National Vulnerability Database/readme.md

```diff
@@ -0,0 +1,38 @@
+# NIST National Vulnerability Database
+The NVD is the U.S. government repository of standards based vulnerability management data represented using the Security Content Automation Protocol (SCAP). This data enables automation of vulnerability management, security measurement, and compliance. The NVD includes databases of security checklist references, security-related software flaws, misconfigurations, product names, and impact metrics. This product uses the NVD API but is not endorsed or certified by the NVD.
+
+
+## Publisher: Paul Culmsee
+
+## Prerequisites
+NIST NVD uses API keys to allow access to the API. You can get an API key [here](https://nvd.nist.gov/developers/request-an-api-key).
+* On the API key requests page, enter data into the three fields on the requests form. 
+* Scroll to the bottom of the Terms of Use, and then click the check box marked "I agree to the Terms of Use."
+* Check the inbox of the email address provided in the steps above for an email from nvd-noreply@nist.gov.
+* Activate and view the API Key by opening the single-use hyperlink. Store the API Key in a secure location as the page will no longer be available after it is closed. If your key is not activated within seven days, a request for a new API Key must be submitted.
+
+## Supported Operations
+
+### Retrieve a collection of CVE
+Get a collection of CVE. The Common Vulnerabilities and Exposures (CVE) program is a dictionary or glossary of vulnerabilities that have been identified for specific code bases, such as software applications or open libraries 
+
+### Retrieve CPE information
+Get a collection of CPE. The Official CPE Dictionary, is a searchable repository of hardware and software products maintained by the National Vulnerability Database (NVD)
+
+## API Documentation
+https://nvd.nist.gov/developers
+
+## Known Issues and Limitations
+* FIltering by date is very specific in terms of format.  It is unlikely you need to filter based on time so I suggest you specify 000 for subseconds and use hh:mm for UTC offset. Eg. For Power Automate flows, this is an example of correctly specifying a date with no UTC offset. formatDateTime(utcNow(), 'yyyy-MM-ddTHH:mm:ss:000 UTC+00:00')
```

Comment:
Readme file should be free from grammar and spelling errors. Please review the file. We recommend using some grammar/spelling checking tools.